src/components/common/Header/index.js

```diff
@@ -36,15 +36,15 @@ const useStyles = makeStyles(theme => ({
   navLinksAnchor: {
```

Comment:
It's good if we keep all our colors in a single file and import them. It would be easy to manage.